evap/evaluation/templates/notebook.html

```diff
@@ -0,0 +1,21 @@
+<div class="collapse position-fixed collapse-horizontal notebook-animation mt-0" tabindex="-1" id="notebook" aria-expanded="false" aria-labelledby="notebook">
+    <div class="card">
+        <div class="card-body notebook-card notebook-div">
+            <div class="offcanvas-header notebook-button mb-2">
+                <h5 class="card-title m-0">
+                    {% trans 'Notebook' %}
+                    <span class="fas fa-circle-info" data-bs-toggle="tooltip" data-bs-placement="right" title="{% blocktrans %}Here you can store private notes that you want to keep ready for future evaluations. The notes will be stored in plain text in your account on the EvaP server, but will not be shown to anyone but you.{% endblocktrans %}"></span>
```

Comment:
please move the icon from the heading to the bottom of the notebook. it better fits on the left side of the button (with a `pe-2`)